src/client/containers/QuizPage/QuizPage.styles.css

```diff
@@ -75,7 +86,17 @@
 }
 
 .radio {
-  font-size: 1.5em;
+  font-size: 1.8em;
+}
+
+.inline {
+  padding-top: 20px;
+  width: 50%;
+  margin-top: 50px;
+  margin-left: 17.44%;
```

Comment:
this seems a bit hacky ? :D

Reply:
done all changes :).. thanks  for review @bytewiz